js/user.js

```diff
@@ -10,7 +10,7 @@ let currentUser;
 /** Handle login form submission. If login ok, sets up the user instance */
 
 async function login(evt) {
```

Comment:
Would be good to rename to something like `loginForm` - I got a bit confused at first why there were 2 `login()` functions - on here, one in model.